packages/morph/tests/dom-helper-test.js

```diff
@@ -56,9 +57,9 @@ test('#setAttribute', function(){
 
   node = dom.createElement('input');
   dom.setAttribute(node, 'disabled', true);
-  equalHTML(node, '<input disabled="true">');
+  equalHTML(node, ie8 ? '<input disabled>' : '<input disabled="true">');
```

Comment:
This can just assert `ok(!!node.getAttribute('disabled'))`.

Then we can avoid the export of `ie8` (we don't detect browsers anywhere else, just features) and use of it only in these tests.